.vscode/settings.json

```diff
@@ -18,5 +18,16 @@
   },
   "[json]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "[handlebars]": {
+    "editor.formatOnSave": false,
+    "editor.formatOnPaste": false
+  },
+  "path-autocomplete.includeExtension": true,
+  "path-autocomplete.extensionOnImport": true,
+  "path-autocomplete.excludedItems": {
+    // do not mess with defaut auto-complete for `js` and `ts`
```

Comment:
I think comments inside the JSON are not a valid syntax.

Reply:
https://code.visualstudio.com/Docs/languages/json#_json-with-comments